src/main/java/CarGameStarter.java

```diff
@@ -0,0 +1,43 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+
+class CarGameStarter {
+  private List<String> carNames = new ArrayList<>();
+  private int trialNum;
+
+  CarGame startGame() {
+    getCarNamesFromUser();
+    getNumOfTrialFromUser();
+    List<Car> cars = carNames.stream()
+        .map(carName -> new Car(carName))
+        .collect(Collectors.toList());
+    return createGame(trialNum, cars);
+  }
+
+  CarGame createGame(int trialNum, List<Car> cars) {
+    return new CarGame(trialNum, cars);
+  }
+
+  void getCarNamesFromUser() {
+    System.out.println("경주할 자동차 이름을 입력하세요.");
+    Scanner scan = new Scanner(System.in);
+    String carNameInput = scan.nextLine();
+    String[] carArr = carNameInput.split(",");
```

Comment:
```suggestion
    carNames = Arrays.asList(carNameInput.split(","));
```
29, 30 라인에 있는 String[] -> List<String> 변환 작업을 위처럼 한 줄로 해볼 수도 있을 것 같네요.